controller/search_blackbox_test.go

```diff
@@ -121,9 +121,28 @@ func (s *searchControllerTestSuite) TestSearchWorkItemsCSV() {
 				return nil
 			}),
 		)
+		// n needs to be >= 4 if parentChilds is set
+		require.True(t, !parentChilds || (parentChilds && n >= 4))
+		if parentChilds {
+			var err error
+			fxtWithLinks, err := tf.NewFixtureIsolated(s.DB, tf.WorkItemLinks(n, func(fxt2 *tf.TestFixture, idx int) error {
```

Comment:
Why do you need the isolated version? We have `tf.WorkItemLinksCustom`.

Reply:
I did not want to create a new fixture creation func, duplicating a lot of code, so I tried to fit the conditional creation of links into that one func. The issue is that if the `tf.WorkItemLinks()` is given as an argument, it will create links, no matter what. Giving it a `num` of `0` does not work. So what is happening there is that it creates the "non-links" fixture first, then adds links if the param required it. Ibrahim and I came up with this solution during a BJN.

Reply:
@kwk We wanted to have a conditional fixture. The Workitem links are created only if the `parentChilds` is set. Using a text fixture would've created all the dependencies. We didn't want to create those since the text fixture before this one had already created the dependencies.